src/main/java/duke/util/UI.java

```diff
@@ -0,0 +1,85 @@
+package duke.util;
+
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.util.Scanner;
+
+/**
+ * A class that provides a simple user interface for the Duke system.
+ */
+public class UI {
```

Comment:
Since UI is an abbreviation, perhaps this could be named Ui instead to follow the recommended coding guidelines